arangopipe/tests/.spyproject/workspace.ini

```diff
@@ -0,0 +1,10 @@
+[workspace]
+restore_data_on_startup = True
+save_data_on_exit = True
+save_history = True
+save_non_project_files = False
+
+[main]
+version = 0.1.0
+recent_files = ['/home/admin2/arangopipe/git_wd2/arangopipe/tests/arangopipe_tests.py', '/home/admin2/arangopipe/git_wd2/arangopipe/arangopipe/arangopipe_admin_api.py', '/home/admin2/arangopipe/git_wd2/arangopipe/tests/arangopipe_testcases.py', '/home/admin2/arangopipe/git_wd2/arangopipe/arangopipe/arangopipe_api.py']
```

Comment:
this looks like it shouldn't be checked in...

Reply:
yes, removed it. We don't need it